src/helics/application_api/FilterOperations.cpp

```diff
@@ -224,8 +224,8 @@ std::shared_ptr<FilterOperator> RandomDelayFilterOperation::getOperator()
 }
 
 RandomDropFilterOperation::RandomDropFilterOperation():
-    tcond(std::make_shared<MessageConditionalOperator>([this](const Message*) {
-        return (randDouble(random_dists_t::bernoulli, (1.0 - dropProb), 1.0) > 0.1);
+    tcond(std::make_shared<MessageConditionalOperator>([this](const Message* /*unused*/) {
+        return (randDouble(RandomDistributions::BERNOULLI, (1.0 - dropProb), 1.0) > 0.1);
```

Comment:
Is the random distribution used for this filter something that should be possible for a user to change?

Reply:
Yes but they would do it via a string, which hasn't changed.  This enumeration definition is not user visible.